pkg/microservice/aslan/core/workflow/handler/webhook.go

```diff
@@ -72,6 +72,11 @@ func processGithub(payload []byte, req *http.Request, requestID string, log *zap
 		log.Errorf("error happens to trigger workflow %v", err)
 		errs = multierror.Append(errs, err)
 	}
-
+	//测试管理webhook
```

Comment:
You can just implement test trigger in the normal github trigger. They are not that different after all.